app/models/manageiq/providers/hawkular/inventory/collector/middleware_manager.rb

```diff
@@ -45,8 +45,11 @@ def config_data_for_resource(resource_path)
       connection.inventory.get_config_data_for_resource(resource_path)
     end
 
-    def metrics_for_metric_type(metric_path)
-      connection.inventory.list_metrics_for_metric_type(metric_path)
+    def metrics_for_metric_type(feed, metric_type_id)
```

Comment:
Right now, it sounds redundant but, if needed, we may add `metrics_for_resource_type` or `metrics_for_resource`, turning ambiguous a `metrics_for_type` method. Could `metrics_with_type` be a better name?